lib/package-detail-view.coffee

```diff
@@ -120,16 +118,16 @@ class PackageDetailView extends View
     @sections.append(new PackageReadmeView(readme))
 
   subscribeToPackageManager: ->
-    @subscribe @packageManager, 'theme-installed package-installed', (pack) =>
+    @packageManager.on 'theme-installed package-installed', (pack) =>
```

Comment:
I was drunk, I hope.

Just forgot about that - I event commented out the `@unsubscribe()` part and commited that :fearful: